### DISCUSSION TITLE
Something about build config

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1262] [Q&A] [answered]
请问rspress是否有某种配置可以设置meta.json中未配置的文件在最终的编译包内不要存在？目前是只要这个md/mdx文件存在，meta.json中未配置 ，最终编译包内还会存在这个html页，且在文档内搜索时会跳转到单独页面内展示此页面内容

**Comment by Timeless0911** [comment, marked as answer]
https://rspress.dev/zh/api/config/config-basic#routeexclude  从路由里排除掉就行了，没必要控制编译配置

**Reply by momo91377** [reply]
感谢，我这边做下尝试

**Reply by momo91377** [reply]
问题已解决，十分感谢
